src/cli/cli.js

```diff
@@ -0,0 +1,34 @@
+import arg from 'arg';
```

Comment:
you could also use https://www.npmjs.com/package/rc, here because we already use it in other places, though not a big deal, because `arg` is already an implicit dependency of delta-chat desktop, so adding it does not make a big difference I think

Reply:
See the edit in my previous comment

Reply:
at the end, I extended our current rc implementation